sambacc/rados_opener.py

```diff
@@ -209,7 +236,17 @@ def enable_rados_url_opener(cls: typing.Type[url_opener.URLOpener]) -> None:
     try:
         import rados  # type: ignore[import]
     except ImportError:
+        _logger.debug("Failed to import ceph 'rados' module")
         return
 
-    _RADOSHandler._rados_api = rados
+    _logger.debug(
+        "Enabling ceph rados support with"
+        f" client_name={client_name!r}, full_name={full_name}"
+    )
+    rados_interface = _RADOSInterface()
+    rados_interface.api = rados
+    rados_interface.client_name = client_name
+    rados_interface.full_name = full_name
```

Comment:
See above comment on `__init__` function. IMHO it would make code more readable here as well.

Reply:
Partly lazyness but also partly that I wanted to highlight the 'internal-ness' of this class. It should only be created inside other code that's already familiar with the structure. Do you still want it?

Reply:
No need. It is just a matter of style.